backend/src/packages/users/user.service.ts

```diff
@@ -1,3 +1,6 @@
+import { SignJWT } from 'jose';
```

Comment:
Let's create a separate jwt service, since we will use it for auth plugin as well.

Reply:
Okay, will do, thank you!